synapse_data_warehouse/synapse/dynamic_tables/V2.21.0__verificationsubmission_latest.sql

```diff
@@ -0,0 +1,38 @@
+use schema {{database_name}}.synapse; --noqa: JJ01,PRS,TMP,CP01
+
+CREATE OR REPLACE DYNAMIC TABLE VERIFICATIONSUBMISSION_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS
+    WITH latest_rows AS (
+        SELECT
+            id AS latest_id,
+            MAX(snapshot_timestamp) AS latest_timestamp
```

Comment:
Is there a change timestamp in this table?

Reply:
Hmm, this is where data size probably comes into the picture, have you taken a look at the prod tables?

Reply:
> Is there a change timestamp in this table?

If you're asking if `CHANGE_TIMESTAMP` is in the final table result from this query, yes it is!

Reply:
Update: I've added `CHANGE_TIMESTAMP` in my CTE in [this commit](https://github.com/Sage-Bionetworks/snowflake/pull/83/commits/0fba27a378afdd1caad138fc2c305897fb23a26b).

See my comment [here](https://github.com/Sage-Bionetworks/snowflake/pull/84#discussion_r1840914306)